### ISSUE TITLE
Jdtls hardcored timestamp...

### ISSUE THREAD

**Issue by PiotrKuzdowicz** [mason-org/mason-registry#5788]
Hey guys, i think hardcoding timestamp of filename is not a good idea.
packages/jdtls/package.yaml
https://github.com/mason-org/mason-registry/blob/8d9759754598c5345f4671573741459b2722ac8f/packages/jdtls/package.yaml#L2

this can cause problems such as https://stackoverflow.com/questions/78297079/how-to-change-the-url-from-where-mason-tries-to-install-a-package

**Comment by Conarius** [comment]
Yeah, but there isn't a quick solution to it as mason will look for exakt file names so it would require a rewrite for mason for that.

**Comment by s1n7ax** [comment]
Who tf though adding date to release artifacts was a good idea :D

**Comment by Conarius** [comment]
> Who tf though adding date to release artifacts was a good idea :D

Don't ask me, ask Eclipse why they do this 😄

**Comment by s1n7ax** [comment]
I have created a custom workflow to re-package jdtls without timestamps. You can add the custom mason registry from nvim-java if you want to use it right now.
https://github.com/nvim-java/release-jdtls

**Comment by Lazerbeak12345** [comment]
related to #8073

**Comment by Lazerbeak12345** [comment]
Timestamp is still present at https://github.com/mason-org/mason-registry/blob/main/packages%2Fjdtls%2Fpackage.yaml#L18

**Comment by Lazerbeak12345** [comment]
the timestamp is still present. the plugin is still outdated. this is still an issue

**Comment by Lazerbeak12345** [comment]
should add that autoclosing is sortof not a great solution. I just had a HORRIBLE finals week in collage, and I didn't even see the notification till just now
